django/contrib/staticfiles/management/commands/collectstatic.py

```diff
@@ -79,7 +79,7 @@ def set_options(self, **options):
         ignore_patterns = options['ignore_patterns']
         if options['use_default_ignore_patterns']:
             ignore_patterns += apps.get_app_config('staticfiles').ignore_patterns
-        self.ignore_patterns = list(set(os.path.normpath(p) for p in ignore_patterns))
+        self.ignore_patterns = list({os.path.normpath(p) for p in ignore_patterns})
```

Comment:
I didn't check it, but do we need to convert it to `list`?

Reply:
Good question. I'll investigate and report back.

Reply:
Yup, any iterable will do. I've removed the outer `list()`.